.appveyor.yml

```diff
@@ -40,7 +43,8 @@ test_script:
       "--------------------------------------------"
       "Running npm run test:without-system-tests..."
       "--------------------------------------------"
-      npm run test:without-system-tests
+  - cmd: npm run %vscode%
```

Comment:
If I understand correctly, this changes allow us to run what was configured in `matrix` in parallel ?

Reply:
Yes, it runs the two different configurations specified in the matrix through `%vscode%